sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -218,5 +218,327 @@ await using (var consumer = client.CreateConsumer(partition, EventPosition.Lates
                 }
             }
         }
+
+        /// <summary>
+        ///   Verifies that the <see cref="EventHubConsumer" /> is able to
+        ///   connect to the Event Hubs service and perform operations.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ReceiveCanReadSingleEvent()
+        {
+            await using (var scope = await EventHubScope.CreateAsync(1))
+            {
+                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
+
+                var eventBatch = new[]
+                {
+                    new EventData(Encoding.UTF8.GetBytes("Lonely"))
+                };
+
+                await using (var client = new EventHubClient(connectionString))
+                {
+                    var partition = (await client.GetPartitionIdsAsync()).First();
+
+                    await using (var producer = client.CreateProducer(new EventHubProducerOptions { PartitionId = partition }))
+                    await using (var consumer = client.CreateConsumer(partition, EventPosition.Latest))
+                    {
+                        // Initiate an operation to force the consumer to connect and set its position at the
+                        // end of the event stream.
+
+                        Assert.That(async () => await consumer.ReceiveAsync(1, TimeSpan.Zero), Throws.Nothing);
+
+                        // Send the batch of events.
+
+                        await producer.SendAsync(eventBatch);
+
+                        // Receive and validate the events; because there is some non-determinism in the messaging flow, the
+                        // sent events may not be immediately available.  Allow for a small number of attempts to receive, in order
+                        // to account for availability delays.
+
+                        var receivedEvents = new List<EventData>();
+                        var index = 0;
+
+                        while ((receivedEvents.Count < eventBatch.Length) && (++index < 3))
```

Comment:
we should probably extract the `3` into a constant now that we're using it in a bunch of places.   I also had to kick it up to `5` when adding the dynamic management stuff, since the hub throughput is now a bit slower with concurrent operations.     Probably would be a good idea to unify on `5`.

Reply:
Made a few more words on this below, based on the same general conversation that we were having in the producer tests.